crates/interpreter/src/exec.rs

```diff
@@ -989,7 +992,11 @@ impl<'m> Thread<'m> {
     }
 
     fn values(&mut self) -> &mut Vec<Val> {
-        &mut self.frame().labels_values
+        &mut self.values
+    }
+
+    fn last_frame_values_cnt(&mut self) -> usize {
+        self.frame().labels.iter().map(|label| label.values_cnt).sum()
     }
```

Comment:
@ia0 The reason why this is not computed on the fly is that I suspect it could be obtained from the side table. Please let me know if this is not the case.

Reply:
Yes, I also believe this could be known statically and doesn't need to be optimized until we get the side table computed. In particular, I expect the `values_cnt` fields to disappear. So all code mentioning `values_cnt` fields should also disappear.